toolkit/tools/imagecustomizerapi/verity.go

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package imagecustomizerapi
+
+import (
+	"fmt"
+)
+
+type VerityPartition struct {
+	IdType IdType `yaml:"IdType"`
+	Id     string `yaml:"Id"`
```

Comment:
You should verify that the value of `Id` has the correct format given the value of `IdType`.

The [isGPTNameValid](https://github.com/microsoft/CBL-Mariner/blob/2.0/toolkit/tools/imagecustomizerapi/partition.go#L90C19-L90C20) function may be useful when checking the `IdTypePartlabel` type.